backend/app/src/strategy/jwt.strategy.ts

```diff
@@ -19,6 +20,18 @@ export class JwtStrategy extends PassportStrategy(Strategy, 'jwt') {
     return jwtPayload;
   }
 
+  private static extractJWTfromHandshake(req: any): string | null {
```

Comment:
type with Express ?

Reply:
Can't type with express, we would need to do a custom type, with not very much added value, I'll do it to get rid of the warning

Reply:
RESOLVED!